myia/info.py

```diff
@@ -19,6 +19,15 @@ def current_info():
 class DebugInfo(types.SimpleNamespace):
     """Debug information for an object.
 
+    If used with the `with` statement, any `DebugInfo` created
+    while inside the `with` body will inherit all attributes:
+
+        with DebugInfo(a=1, b=2):
```

Comment:
Code snippets in docstrings should probably use `>>>` and `...` so that they are compatible with e.g. [`doctest`](https://docs.python.org/3.6/library/doctest.html) and Sphinx.

Reply:
Ah, right.